lib/lrama/grammar/parameterizing_rule_resolver.rb

```diff
@@ -17,7 +17,7 @@ def build_rules(token, rule_counter, lhs_tag, line)
         builder = @parameterizing_rule_builders.select { |b| b.name == token.s_value }.last
         raise "Unknown parameterizing rule #{token.s_value} at line #{token.line}" unless builder
 
-        builder.build_rules(token, rule_counter, lhs_tag, line)
+        builder.build_rules(token, token.args, rule_counter, lhs_tag, line, @parameterizing_rule_builders)
```

Comment:
[note] At the moment, `parameterizing_rule_builders` is passed. But it might be changed to pass resolver to share rules once it's created.